Fix minor grammatical error (#4098)
rust-lang/book: src/ch05-03-method-syntax.md

>
> Rust doesn’t have an equivalent to the `->` operator; instead, Rust has a
> feature called *automatic referencing and dereferencing*. Calling methods is

> > one of the few places in Rust that have this behavior.

Reviewer: ```suggestion
> one of the few places in Rust with this behavior.
```
